packages/repository/src/crud-connector.ts

```diff
@@ -32,7 +32,7 @@ export interface CrudConnector extends Connector {
   find(modelClass: Class<Entity>, filter?: Filter, options?: Options):
     Promise<EntityData[]>;
 
-  findById?(modelClass: Class<Entity>, id: AnyType, options?: Options):
+  findById?(modelClass: Class<Entity>, id: Any, options?: Options):
```

Comment:
It makes me wonder whether TypeScript provides a construct allowing us to specify that `id` should be of the same type as the primary key of `Class<Entity>`? If this cannot be achieved, then I think it's better to use `any` type and disable `no-any` check for this entire class.

My goal with `no-any` rule was to make us think harder before using it, but in no way I wanted to get into situation where we just replace `any` with `Any`.